### PR TITLE
Switch to using prebuilt docker image

### PR DESCRIPTION
**Pull request by mihails-strasuns-sociomantic**

### CODE REVIEW

**Comment by mihails-strasuns-sociomantic** [comment]
@leandro-lucarella-sociomantic seems to work and total test time is 3x less (5 minutes vs 15 of random other PR I checked). Shall we proceed?

**Comment by leandro-lucarella-sociomantic** [comment]
Right, this seems to take 10 minutes off the build time, not bad.

**Comment by leandro-lucarella-sociomantic** [comment]
But still I want to get the scripts somewhere before proceeding. Give me a few more days please.

**Comment by mihails-strasuns-sociomantic** [comment]
Looking forward it!

**Comment by mihails-strasuns-sociomantic** [comment]
Rebased on v2.6.x branch and modified to use https://hub.docker.com/r/sociomantictsunami/dlang @leandro-lucarella-sociomantic

**Comment by mihails-strasuns-sociomantic** [comment]
Fails because of missing ocean dependencies, need to figure out good way to update them too in context of travis matrix

File: .travis.yml [hunk 1]
```diff
@@ -7,8 +7,5 @@ services:
 # a lightweight environment
 language: c
 
-install:
-    - ci/travis-install.sh
```
Comment (leandro-lucarella-sociomantic) [inline .travis.yml:11]: This is still needed! This code is the one in charge to build an image specifically for Ocean (based on the `dlang` image).

**Review by leandro-lucarella-sociomantic — COMMENTED** [review]
OK, this is incorrect because you are removing a lot of needed travis boilerplate (which will be "solved" soon). I will make an alternative PR, I think it will be easier this way.

**Comment by mihails-strasuns-sociomantic** [comment]
Closing in favor of https://github.com/sociomantic-tsunami/ocean/pull/90

**Pull request closed without merging**